kotlinx-coroutines-core/common/src/flow/operators/Chunk.kt

```diff
@@ -0,0 +1,64 @@
+package kotlinx.coroutines.flow.operators
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.flow
+
+
+fun <T, R> Flow<T>.chunked(size: Int, transform: suspend (List<T>) -> R): Flow<R> = nonOverlappingWindowed(size, size, true, transform)
```

Comment:
Shouldn't this default to an identity transform?

Reply:
Yes, done.